.github/workflows/testBuildHybrid.yml

```diff
@@ -3,16 +3,19 @@ name: Build and deploy hybrid apps for testing
 on:
   workflow_dispatch:
     inputs:
+      # Pull Request number from App repo for correct placement of ND app. If not specified defaults to main branch.
       APP_PULL_REQUEST_NUMBER:
-        description: Pull Request number from App repo for correct placement of ND app. If not specified defaults to main branch.
+        description: App PR number. Defaults to main.
         required: false
         default: ''
+      # Pull Request number from Mobile-Expensify repo for correct placement of OD app. It will take precedence over MOBILE-EXPENSIFY from App's PR description if both specified. If nothing is specified defaults to Mobile-Expensify's main
```

Comment:
```suggestion
      # Pull Request number from Mobile-Expensify repo for correct placement of OD app. It will take precedence over MOBILE-EXPENSIFY from App's PR description if both are specified. If nothing is specified defaults to Mobile-Expensify's main
```

Reply:
Actually

```suggestion
      # Pull Request number from Mobile-Expensify repo for correct placement of OD app. It will take precedence over MOBILE-EXPENSIFY from App's PR description if both are specified. If nothing is specified, it defaults to Mobile-Expensify's main
```